### PR TITLE
Translations Update from Weblate

### PR DESCRIPTION
**Pull request by UptimeKumaBot**
Please go to https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/ if you want to contribute too.🐻

If there is anything not clear on Weblate, feel free to ping me (@louislam) here. I can add some screenshots or explanations on Weblate. 

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by CommanderStorm** [comment]
@louislam 
About the translation effort:
Do you think ordering some translations (only for languages which have no current siginificant translation) from the https://opencollective.com/uptime-kuma would be possible?
From a cost perspective, I think this would not be that bad (~€40-200/language).

- [albanian](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/sq/): https://www.fiverr.com/kledjona/translate-english-italian-greek-and-french-to-albanian
- [bengali](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/bn/): https://www.fiverr.com/wordtranslators/translate-japanese-korean-chinese-thai-vietnamese-languages
- [punjabi](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/pa/): https://www.fiverr.com/laeeque009/translation-in-urdu-to-hindi-and-hindi-to-urdu-english-to-hindi-urdu-punjabi
- [hindi](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/hi/): https://www.fiverr.com/laeeque009/translation-in-urdu-to-hindi-and-hindi-to-urdu-english-to-hindi-urdu-punjabi
- [Catalan](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/ca/): https://www.fiverr.com/qualitywriter_s/do-the-best-translation-english-spanish-french-catalan


I also think we might want to consider removing some local translation choices **if there is no significant translation effort**:
- [Panjabi](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/pa/) vs [Panjabi (Pakistan)](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/pa_PK/) => proposal: delete `pa_PK` as both have no significant effort put into them
- [Hebrew](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/he/) vs [Hebrew (Israel)](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/he_IL/) => proposal: delete `he` as it does have no significant effort put into it, rename `he_IL` -> `he`

I am unsure about these ones (if somebody knows something about these...):
- [Serbian (latin)](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/sr_Latn/) vs [Serbian](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/sr/)

**Comment by chakflying** [comment]
I think I'm not against this, but I also don't see a strong need. I think the real user-facing part of this application is only the status pages, whereas the dashboard is aimed at developers/IT admin with at least some technical knowledge, where English is the preferred language for accuracy in technical communication. The vast majority of translation keys are only used in the dashboard, where I think it's not as important that all keys are translated.

But I'm just one perspective, maybe in other places English is not as commonly used in IT.

**Comment by louislam** [comment]
Also prefer to be translated by contributors, because I believe contributors here should be using Uptime Kuma, they know the context of those words better. 

For my experience, those paid translators are good at translating an article or subtitles. But for software, since a lot of words are just a single word, they don't really try to install it and understand the context of all features before translation.

For those incomplete (less than 1%) translations, those json files were created via Weblate. They don't appear in the language list, as they have to be added into the array manually. I cleared some before, but they could be back if someone create them on Weblate. So just keeping them there should be fine.

**Comment by mh166** [comment]
> I am unsure about these ones (if somebody knows something about these...):
> 
> * [Serbian (latin)](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/sr_Latn/) vs [Serbian](https://weblate.kuma.pet/projects/uptime-kuma/uptime-kuma/sr/)

Both of these are the same Serbian language. They just use two different alphabets actually: a (mostly) Latin alphabet with some additional characters. And the Cyrillic alphabet (which people usually connect with Russian texts).

I'm aware of the two different alphabets, because some friends of mine speak serbian. That being said, I don't know much about the Serbian people's preferences. Therefore, please find a quote about it [from Wikipedia](https://en.wikipedia.org/wiki/Romanization_of_Serbian#:~:text=Standard%20Serbian%20uses%20both%20alphabets%20currently.%20A%20survey%20from%202014%20showed,the%20remaining%2017%25%20preferred%20neither.) instead:
> Standard Serbian uses both alphabets currently. A survey from 2014 showed that 47% of the Serbian population favors the Latin alphabet whereas 36% favors Cyrillic; the remaining 17% preferred neither.

In conclusion, both of these translations are quite interchangeable and probably either is understood by the vast majority of people. But maybe _because_ it is not a huge effort (for a native speaker) to translate either language, both of them should be kept to allow for personal preference and ease of reading.

**Comment by kindercat** [comment]
Hi guys!

I made some corrections for the Romanian language and although they appear in the PR (now I see that it has been moved to 2.0), I do not see the changes in live version.
The changes will be visible when version 2.0 is released?

**Review by CommanderStorm — APPROVED** [review]
Changes look fine to me. Merging as weblate does not seem to like being this far behind master as stated in https://github.com/louislam/uptime-kuma/issues/4788
